src/vs/workbench/browser/parts/editor2/nextEditorDragAndDrop.ts

```diff
@@ -156,7 +156,7 @@ class DropOverlay extends Themable {
 			// Close in source group unless we copy
 			const copyEditor = this.shouldCopyEditor(draggedEditor, event);
 			if (!copyEditor) {
-				sourceGroup.closeEditor(draggedEditor.editor);
+				sourceGroup.closeEditor(draggedEditor.editor, true);
```

Comment:
I think we can tweak this a little bit: If the user drags an editor to another group and drops it there, I would think that it is fine to close the editor if its the last. Same for dragging it to another group to split the group there. 

In other words, maybe the only time we should allow the editor group to stay empty is when you drag the editor into the same group it originates from and you are splitting that group into one of the directions.